src/List/Title/__snapshots__/Title.component.spec.tsx.snap

```diff
@@ -24,19 +10,34 @@ exports[`Component: Title should be defined 1`] = `
   font-weight: 500;
   text-align: inherit;
   color: inherit;
-  color: #808080;
   font-weight: 700;
 }
 
 .c1 small {
   font-size: 87.5%;
 }
 
+.c0 {
+  box-sizing: border-box;
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  padding: 0 1rem 0;
+  -webkit-align-items: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  height: 2rem;
+  margin-top: 0.75rem;
+}
+
 <div
   className="c0 anchor-list-title"
 >
   <span
     className="c1 anchor-typography"
+    color="text.label"
```

Comment:
But this should be gone (see `$color` vs `color` comment)

Reply:
It is now, thanks!